ci/awslogs.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+set -eu
+
+GLOBAL_LAST_UPDATE=0
+
+# in seconds
+ALERT_THRESHOLD=3600
+STOP_THRESHOLD=5400
+
+# monitor these log grouips
+cat <<EOF > /tmp/monitor_loggroups
+kubernetes-development
+kubernetes-production
+kubernetes-staging
+this-log-group-will-alert-cnelson
```

Comment:
Is it useful to keep something like this around for testing, or should we drop it?

Reply:
I'll delete it before we merge, I just left it in for testing so I could validate this was working